model/patch_lifecycle.go

```diff
@@ -993,36 +984,20 @@ func AbortPatchesWithGithubPatchData(ctx context.Context, createdBefore time.Tim
 			continue
```

Comment:
Can't comment on it, but the doc comment on `AbortPatchesWithGithubPatchData` should remove mention of the legacy commit queue.